README.md

```diff
@@ -38,6 +39,11 @@ To run tests locally, run `make test`. Please don't push to master with
 failing tests—Jenkins will refuse to deploy your code, and nobody will be able
 to deploy until fixing it.
 
+If you make a pull request to the OCF GitHub organization from your fork of
+ocfweb, Jenkins will attempt to build your branch, and if your build fails,
+you can log into [Jenkins](https://jenkins.ocf.berkeley.edu) with your OCF
+account credentials to see which tests you've failed and fix them. Almost
+invariably it's something `pre-commit` would have caught.
```

Comment:
There are quite a few tests on this repository, so I'd just remove this sentence altogether. I think it assumes the changes being made are small, which isn't necessarily the case, and there is a more detailed section directly below about `pre-commit` and the git hooks it installs.